src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -104,21 +103,26 @@ const SubsidiesSummary = ({
                 couponCodeRequestsCount={couponCodeRequests.length}
                 totalCoursesEligibleForCertificate={totalCoursesEligibleForCertificate}
                 programProgressPage={programProgressPage}
-                className="mb-3"
+                className="mb-2 border-remove"
               />
             )}
-            {searchCoursesCta}
+            {canEnrollWithEnterpriseOffers
+           && (
```

Comment:
tiniest of nits: combine these 2 lines (e.g., `{canEnrollWithEnterpriseOffers && (`) to improve the readability a bit